### PR TITLE
doc: Added Application Groups Doc

### PR DESCRIPTION
**Pull request by ashokdevtron**

### CODE REVIEW

**Review by abhibhaw — CHANGES_REQUESTED** [review]
RBAC is missing, who can create filters, who can trigger builds, what if I don't have same permission on all applications etc.

**Comment by ashokdevtron** [comment]
> RBAC is missing, who can create filters, who can trigger builds, what if I don't have same permission on all applications etc.

Done 👍

**Review by abhibhaw — DISMISSED** [review]
@uxarya-d Please confirm on the RBAC

File: docs/user-guide/application-groups.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+1. In the **Build & Deploy** tab of your application group, select the intended applications and click the **Change Branch** button present at the bottom.
+
+    ![Figure 20: Changing Branch](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/application-groups/change-branch.jpg)
+
+2. Enter the new branch name (*or the new regular expression if you are using branch regex*) and click **Update Branch**.
+
+    ![Figure 21: Updating Branch Name](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/application-groups/update-branch.jpg)
+
+{% hint style="info" %}
+Users need to have [Manager role](../user-guide/global-configurations/authorization/user-access.md#role-based-access-levels) or above (along with access to the environment and applications) to update their branch
```
Comment (uxarya-d) [inline docs/user-guide/application-groups.md:164]: Admin role and above…

**Review by uxarya-d — CHANGES_REQUESTED** [review]
Pls check comments

**Review by uxarya-d — APPROVED** [review]

**Pull request merged**